xml/release-notes.xml

```diff
@@ -311,6 +311,27 @@
 
   </sect2> -->
 
+  <sect2 xml:id="sec.upgrade.kernel">
```

Comment:
Make dashes not hyphens (admittedly, that is new since I redid the Leap 15.3 RN):
```suggestion
  <sect2 xml:id="sec-upgrade-kernel">
```